### PR TITLE
remove learn, update readme

### PR DESCRIPTION
**Pull request by praveenweb**
### Description

This PR removes `community/learn` from the repo. The contents will be moved to a new repo [hasura/learn-graphql](https://github.com/hasura/learn-graphql).
Updated relevant readme and contributing guide.
Transferred the open issues related to learn.

### Affected components 

- Community Content

### CODE REVIEW

File: .netlify/state.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (shahidhk) [inline .netlify/state.json]: @praveenweb Is this file required?

Comment (praveenweb) [reply .netlify/state.json]: It's Netlify deployment of `graphiql-online`. Removed now.
